lib/chat_app_web/live/home_live.ex

```diff
@@ -37,4 +42,35 @@ defmodule ChatAppWeb.HomeLive do
     </div>
     """
   end
+
+  def handle_event(
+        "create_room",
+        %{
+          "room_name" => room_name,
+          "owner_name" => owner_name,
+          "is_private" => is_private,
+          "password" => password
+        },
+        socket
+      ) do
+    is_private =
+      if is_private == "on" do
+        true
+      else
+        false
+      end
+
+    room = %{
+      "room_name" => room_name,
+      "owner_name" => owner_name,
+      "max_participants" => 20,
+      "is_private" => is_private,
+      "password" => password,
```

Comment:
Maybe we could use hash room passwords later on 😝

Reply:
Yeah, but for now I just wanted to have it working, will add as an idea